lib/schwifty-models/tokens.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const Uuid = require('uuid/v4');
+const Joi = require('joi');
+const Model = require('schwifty').Model;
+
+module.exports = (srv, options) => {
```

Comment:
Haute has been updated, so no particular need to wrap the model classes in a function anymore.  Unless `srv`/`options` are really necessary!